.env.local.sample

```diff
@@ -1,5 +0,0 @@
-AUTH0_SECRET='use [openssl rand -hex 32]'
```

Comment:
is this needed for this issue?

Reply:
I have renamed .env.local.sample to .env.local to make auth0 work. So it is showing that .env.local.sample is deleted.